openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -111,10 +109,20 @@ def recurring_nudge_schedule_bin(
             language,
             context,
         )
-        _recurring_nudge_schedule_send.apply_async((site_id, str(msg)), retry=False)
+        with newrelic_function_trace('enqueue_send_task'):
+            _recurring_nudge_schedule_send.apply_async((site_id, str(msg)), retry=False)
+
+
+def _annotate_newrelic_transaction(message_type, site, bin_num, target_day_str):
```

Comment:
Nit: Maybe remove `newrelic` from the function name here? The reason `monitoring_utils` was created was to try to insulate the rest of the code from New Relic specifics in case we switch APM solutions.

Reply:
Please document in the comments what these things represent and what types they are. Also, please add that information to: https://openedx.atlassian.net/wiki/spaces/PERF/pages/54362736/Custom+Metrics+in+New+Relic

Reply:
Added comments, I'll update that page when this merges!